src/egithub_webhook.erl

```diff
@@ -73,30 +75,58 @@ write_comments(Cred, Repo, PR, Comments, Messages) ->
           position  := Position,
           text      := Text
          }) ->
-      write_comment(Cred, Repo, PR, CommitId, Path, Position, Text, Comments)
+            write_line_comment(Cred, Repo, PR, CommitId,
+                               Path, Position, Text, Comments);
+       (#{text := Text,
+          position := 0
```

Comment:
Shouldn't this be the first clause in the fun? I'm pretty sure the previous one matched if ``Position == 0``